docs/management/managing-licenses.asciidoc

```diff
@@ -0,0 +1,21 @@
+[[managing-licenses]]
+== License Management
+
+When you install Kibana, it generates a Basic license
+with no expiration date.  You can see the status of your
+license by opening *Management* and then clicking *License
+Management*.
+
```

Comment:
I'd say something about being able to update your license and start a trial up front. Maybe:

To start a 30-day trial, view the status of your license, or install a new license go to *Management* and click *License Management*.